File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.java
// Last updated: 17/07/2025, 11:25:55
class Solution {
    public int[] runningSum(int[] nums) {
        int[] ans = nums;
        for(int i=1; i<nums.length; i++){
            ans[i] += ans[i-1];
        }
        return ans;
    }
}

File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.py
# Last updated: 17/07/2025, 11:25:50
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        running_sum = []
        
        for i in range(0,len(nums)):
            running_sum.append(nums[i]+sum(nums[0:i]))
        
        return running_sum